### PR TITLE
Port Moonbeam launch related changes

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/223
https://github.com/PureStake/moonbeam-docs/pull/230
https://github.com/PureStake/moonbeam-docs/commit/bba3ec16670ade4943f41e0a154dfd2bbbc7491e

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: learn/features/governance.md [hunk 1]
```diff
@@ -26,6 +26,34 @@ Moonbeam作为去中心化网络，将由核心开发者、应用程序开发者
  - **锁定期** —— 赢得投票用户的代币锁定期（提案执行后）。在此期间，用户仍可使用锁定代币进行质押或投票
  - **委托** —— 将自己的投票权委托给其他账户，以积累一定信念值的行为
 
+=== "Moonbeam"
+    |         变量         |  |                                                            Value                                                             |
```
Comment (eshaben) [inline learn/features/governance.md:30]: should value be translated?

File: learn/features/treasury.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ description: 作为波卡（Polkadot）平行链，Moonbeam将使用由理事会
 
 目前，财政库的价值如下：
 
+=== "Moonbeam"
+    |            变量             |  |                                                        Value                                                         |
```
Comment (eshaben) [inline learn/features/treasury.md:30]: same comment here as above - should value be translated?

File: learn/features/governance.md (hunk 1 above)
Comment (eshaben) [reply learn/features/governance.md:30]: same goes for the Moonriver and Moonbase Alpha

File: tokens/governance/proposals.md [hunk 3]
```diff
@@ -34,14 +34,30 @@ description: 如何通过治理机制在Moonbeam上发送提案
...
-目前Moonbase Alpha有关参数设置如下：
-
-|      变量      |      |                              值                              |
-| :------------: | :--: | :----------------------------------------------------------: |
-|     发起期     |      | {{ networks.moonbase.democracy.launch_period.blocks}} blocks ({{ networks.moonbase.democracy.launch_period.days}} days) |
-|     冷却期     |      | {{ networks.moonbase.democracy.cool_period.blocks}} blocks ({{ networks.moonbase.democracy.cool_period.days}} days) |
-| 最低原像存款额 |      |    {{ networks.moonbase.democracy.min_preim_deposit}} DEV    |
-| 最低提案存款额 |      |       {{ networks.moonbase.democracy.min_deposit}} DEV       |
+=== "Moonbeam"
+    |         变量         |  |                                                          Value                                                          |
```
Comment (eshaben) [inline tokens/governance/proposals.md:38]: same comment here as before... should value be translated?

File: tokens/governance/voting.md [hunk 4]
```diff
@@ -35,13 +35,27 @@ description: 如何通过Moonbeam治理机制来投票决定执行或否决提
  - **锁定期** —— 赢得投票用户的代币锁定期（提案执行后）。在此期间，用户仍可使用锁定代币进行质押或投票
  - **委托** —— 将自己的投票权委托给其他账户，以积累一定信念值的行为
 
-目前Moonbase Alpha有关参数设置如下：
+=== "Moonbeam"
+    |       变量         |  |                                                         Value                                                         |
```
Comment (eshaben) [inline tokens/governance/voting.md:39]: same comment here about value

**Review by eshaben — CHANGES_REQUESTED** [review]
Ahh so many changes haha sorry to put this all in one PR! 😅 

It looks like there were some pages that were missed... 
-`builders/integrations/indexers/thegraph.md` (https://github.com/PureStake/moonbeam-docs/pull/223/files#diff-fd7e81c6b9974ef19edecab585c2900c991b8b59b331954e5de5d812e55911feR18-R156)
- builders/tools/eth-dev-env/hardhat.md (https://github.com/PureStake/moonbeam-docs/pull/223/files#diff-dc04a14539436f48bfe27667c2ed9e60ab0f2d8d369c78a4228f5a6fd6002b17R34-R73)
- learn/platform/tokens.md (https://github.com/PureStake/moonbeam-docs/pull/223/files#diff-fdaa7573114446474b9a2a0616c6e7488e0abbc4b1090fd7a21516e0814a2e9cR17-R27) 👈 these seem to be minor just not sure if it makes a difference 🤷‍♀️ 
- node-operators/indexer-nodes/thegraph-node.md (https://github.com/PureStake/moonbeam-docs/pull/223/files#diff-f31bdb0b960f43552a5a75f03c917375ee30034037933814d7d9664c7fee06dbR26-R127) 👈  Is this whole page out of date? It seemed to be... if there is a lot to translate we can open a different ticket for this. whatever you think is best!

File: builders/tools/eth-libraries/web3js.md [hunk 5]
```diff
@@ -29,8 +29,25 @@ const web3 = new Web3('RPC_URL');
...
+    ```
+    {{ networks.moonriver.rpc_url }}
+    ```
+
+=== "Moonbase Alpha"
+    ```
+    {{ networks.moonbase.rpc_url }}
+    ```
+
+=== "Moonbeam Dev Node"
```
Comment (eshaben) [inline builders/tools/eth-libraries/web3js.md:47]: This should be Moonbeam开发节点 I think

File: builders/tools/eth-libraries/web3py.md [hunk 6]
```diff
@@ -28,8 +28,25 @@ web3 = Web3(Web3.HTTPProvider('RPC_URL'))
...
+    ```
+    {{ networks.moonriver.rpc_url }}
+    ```
+
+=== "Moonbase Alpha"
+    ```
+    {{ networks.moonbase.rpc_url }}
+    ```
+
+=== "Moonbeam Dev Node"
```
Comment (eshaben) [inline builders/tools/eth-libraries/web3py.md:46]: This should be Moonbeam开发节点 I think

File: builders/tools/sidecar.md [hunk 7]
```diff
@@ -39,14 +39,19 @@ node_modules/.bin/substrate-api-sidecar --version
...
     export SAS_SUBSTRATE_WS_URL=wss://wss.moonriver.moonbeam.network
     ```
 
 === "Moonbase Alpha"
     ```
-    export SAS_SUBSTRATE_WS_URL=wss://wss.testnet.moonbeam.network
+    export SAS_SUBSTRATE_WS_URL=wss://wss.api.moonbase.moonbeam.network
     ```
 
 === "Moonbeam Dev Node"
```
Comment (eshaben) [inline builders/tools/sidecar.md:57]: This should be Moonbeam开发节点 I think

**Review by eshaben — CHANGES_REQUESTED** [review]

File: learn/features/governance.md (hunk 1 above)
Comment (eshaben) [reply learn/features/governance.md:30]: I would maybe do a search in the repo for all use cases of "Variable" and Value" and update those cause I actually just noticed that "Variable" isn't translated here https://github.com/PureStake/moonbeam-docs-cn/pull/44/files#diff-c9c61564fa95109849566c20d6079bb70faa093f2ccb926fad1ad3a43ef123c9R39-R45

**Comment by hyd628** [comment]
Thanks for the feedback @eshaben . 

builders/integrations/indexers/thegraph.md was updated in a separate PR and merged: https://github.com/PureStake/moonbeam-docs-cn/commit/9dade795f1edd6f4a92d228cf4301dcc36190823

builders/tools/eth-dev-env/hardhat.md: missed this because there was another hardhat file, should be updated now.

learn/platform/tokens.md: the tense changes dont need to be translated (tenses are usually not explicit in Chinese), but I did find another unrelated issue, which I edited. 

node-operators/indexer-nodes/thegraph-node.md : updated this file. missed it because i just skipped anything related to the graph due to the other PR opened. 

Thanks!

File: node-operators/indexer-nodes/thegraph-node.md [hunk 8]
```diff
@@ -54,9 +49,25 @@ git clone https://github.com/graphprotocol/graph-node/ \
...
+    ```
+    ethereum: 'moonriver:{{ networks.development.rpc_url }}'
+    ```
+
+=== "Moonbase Alpha"
+    ```
+    ethereum: 'mbase:{{ networks.development.rpc_url }}'
+    ```
+
+=== "Moonbeam Dev Node"
```
Comment (eshaben) [inline node-operators/indexer-nodes/thegraph-node.md:67]: this should be "Moonbeam开发节点"

**Review by eshaben — CHANGES_REQUESTED** [review]

**Comment by eshaben** [comment]
@hyd628 

So it looks like not all of the updates were made on `builders/integrations/indexers/thegraph.md` - I know you updated it because it was out of date, but then it was updated again for the launch of Moonbeam and it seems like there were places where Moonbeam was added in but also a few places its missing:

- Moonbeam is not listed in the quick start section: [English changes](https://github.com/PureStake/moonbeam-docs/pull/223/files#diff-fd7e81c6b9974ef19edecab585c2900c991b8b59b331954e5de5d812e55911feR22-R28) and [Chinese changes missing](https://github.com/PureStake/moonbeam-docs-cn/commit/9dade795f1edd6f4a92d228cf4301dcc36190823#diff-fd7e81c6b9974ef19edecab585c2900c991b8b59b331954e5de5d812e55911feR24-R40)
- Checking Prerequisites section is missing the "can be adapted for Moonbeam or Moonriver" piece: [English changes](https://github.com/PureStake/moonbeam-docs/pull/223/files#diff-fd7e81c6b9974ef19edecab585c2900c991b8b59b331954e5de5d812e55911feR52) and [Chinese changes missing](https://github.com/PureStake/moonbeam-docs-cn/commit/9dade795f1edd6f4a92d228cf4301dcc36190823#diff-fd7e81c6b9974ef19edecab585c2900c991b8b59b331954e5de5d812e55911feR46)
- Subgraph Manifest section is missing part about Moonbeam and Moonriver: [English changes](https://github.com/PureStake/moonbeam-docs/pull/223/files#diff-fd7e81c6b9974ef19edecab585c2900c991b8b59b331954e5de5d812e55911feR153) and [Chinese changes missing](https://github.com/PureStake/moonbeam-docs-cn/commit/9dade795f1edd6f4a92d228cf4301dcc36190823#diff-fd7e81c6b9974ef19edecab585c2900c991b8b59b331954e5de5d812e55911feR147)


I also happened to notice that I missed this page... it looks like `builders/tools/debug-trace.md` hasn't been updated: [English changes](https://github.com/PureStake/moonbeam-docs/pull/223/files#diff-274667756a18a6d0f20bb7a8f2aecce0f433538428ddcbddc45eb121f3d9e298R68-R115)

**Comment by hyd628** [comment]
@eshaben incorporated all the feedback. Thanks!

**Review by eshaben — APPROVED** [review]
LGTM ✨

**Pull request merged**